contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -1,5 +1,14 @@
 # Authenticate API requests
 
+> [!CAUTION]
```

Comment:
Probably should use `::note` annotations as they look better. You can do warnings as well https://github.com/backstage/backstage/pull/24473

Reply:
is that the same here in contrib rather than the docs folder?

Reply:
Ah I didn't catch that this was not docs, but yeah, you can do it in github markdown too.

Reply:
Actually scrap that - what you have is good.